src/components/ground-nav/ground-nav.scss

```diff
@@ -0,0 +1,262 @@
+@use '../../design-tokens/breakpoint.yml';
+@use '../../design-tokens/colors.yml';
+@use '../../design-tokens/font.yml';
+@use '../../mixins/ms';
+@use '../../mixins/unit';
+
+// Whitespace used for gaps, padding and margin
+// (It uses `rem` so space will be consistent throughout the component)
+$_ground-nav-space: ms.step(1, 1rem);
```

Comment:
FYI: The way Sass variables work, if you prefix them with a leading underscore, they won't leak out into other files as long as those files import the variable with `@use`.

So if you want to be a bit more succinct with them in the context of a component file, you can. No specific action item required.